src/server/game.class.js

```diff
@@ -0,0 +1,14 @@
+// class Game
+
+module.exports = class Game {
+
+    constructor(gameID) {
+        this.id = gameID;
+        this.persona = undefined;
+        this.players = [];
```

Comment:
There is the spectate use case. But I don't think it makes sense to add the name and Ids of spectators and  broadcast it to everybody watching.

Reply:
Then I suggest we change the plans to not use the ``room`` message to update the game state, but create a new message called ``game`` which holds only the game information. Then the states are handled separately (and I don't have to include  possibly duplicate arrays every time)

And maybe it would be better to put the votes/answers information on the user who provided them, eliminating the votes/answer arrays.

About "including" information, the idea is the opposite. To not include things, I have to actively remove them from the object. But I agree, it doesn't make sense to inform the users id or room, so I'll remove these.